src/main/resources/hudson/plugins/logparser/LogParserAction/index.jelly

```diff
@@ -4,7 +4,7 @@
   <j:set var="parseSucceeded" value="${it.result.failedToParseError == null}"/>
   <j:set var="parseFailed" value="${it.result.failedToParseError != null}"/>
   <j:set var="badParseRules" value="${it.result.badParsingRulesError != null}"/>
-  <l:layout>
+  <l:layout title="Parsed Console Output">
```

Comment:
This is unrelated to CSP, but title is required on `layout` tag so I added it. Can revert if requested.

Reply:
thx for the MR, fine for me